src/models/EventPayload.ts

```diff
@@ -0,0 +1,79 @@
+import { Action } from './chat/Action';
+
+/**
+ * The payload accepted by the Analytics Events API.
+ *
+ * @public
+ */
+export interface EventPayload {
+  /** The user action which caused the event, e.g. ADD_TO_CART or THUMBS_UP  */
+  action: Action;
+  /** Unique identifier to tie together events in a single browsing session */
+  sessionId?: string;
+  /** The URL of the page where the event occurred */
+  pageUrl?: string;
+  /** The URL of the page the event is directing the visitor to. */
+  destinationUrl?: string;
+  /** A label assigned to the event, e.g. a CTA label. */
+  label?: string;
+  /** The locale of the user who generated the event. */
+  locale?: string;
+  /** The timestamp at which the event occurred, in ISO format. */
+  timestamp?: Date | string;
```

Comment:
Curious if we tested this.

Reply:
when doing `JSON.stringify` on a Date object, it looks to default to ISO format already.

<img width="408" alt="Screenshot 2023-06-27 at 11 35 11 AM" src="https://github.com/yext/analytics/assets/36055303/8e65e372-709a-437e-b282-336062f26f41">

![Screenshot 2023-06-27 at 11 35 19 AM](https://github.com/yext/analytics/assets/36055303/869c4054-aeaa-4894-91e0-b47f4db0778c)